<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSelect</class>
 <widget class="QWidget" name="PanelSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Панель:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCopy">
     <property name="text">
      <string>Копировать</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
